templates/hosted-engine.conf.j2

```diff
@@ -25,6 +25,15 @@ ca_subject="C=EN, L=Test, O=Test, CN=Test"
 vdsm_use_ssl=true
 gateway={{ he_gateway }}
 bridge={{ he_mgmt_network }}
+{% if he_network_test is defined %}
```

Comment:
I suggest to remove this initializing he_network_test to 'dns' in defaults/main.yml in order to implement the new behavior also for who is going to directly consume the role without the CLI wrapper.
Please also document it in /README.md

Reply:
good idea, done